src/create_namespace.ts

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios'
+
+export async function createNamespace(
+  name: string,
+  guild: string,
+  url: string
+): Promise<string> {
+  const {data} = await axios.post(url, {
```

Comment:
It'd probably make sense to check the response status code and make github action failed in case of code is not 201.